.github/workflows/dependabot-auto-evaulate-pr.yaml

```
@@ -0,0 +1,31 @@
+name: Dependabot Evaluation
+
+on:
+    pull_request:
+        paths:
+            - "cookbook/**/*"
+
+jobs:
+    evaluate_dependabot_pr:
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v2
+
+            - name: Set up Python
+              uses: actions/setup-python@v2
+              with:
+                  python-version: 3.x
+
+            - name: Install dependencies
+              run: |
+                  python -m pip install --upgrade pip
+                  pip install PyGithub marvin
```

just so this works with Marvin 2.x even if we only have the pre-release out now
```suggestion
                  pip install PyGithub git+https://github.com/PrefectHQ/marvin.git
```